test/Dapr.E2E.Test/ServiceInvocation/InvokeMethodTest.cs

```diff
@@ -0,0 +1,30 @@
+namespace Dapr.E2E.Test
+{
+    using System;
+    using System.Net.Http.Json;
+    using System.Text.Json;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Dapr.Client;
+    using Xunit;
+
+    public class Main
+    {
+        [Fact]
+        public async void TestServiceInvocation()
+        {
+            var testApp = new DaprApp("testapp", 5000);
```

Comment:
Make this implement `IDisposable` and do `using var testapp ...`

Reply:
It might be better to auto-assign the port for the APP too. You can configure the listening address for ASP.NET Core like `--urls http://localhost:5000`